package.json

```
@@ -3,21 +3,18 @@
   "version": "0.1.2",
   "scripts": {
     "build": "npm run build:esm && npm run build:cjs",
-    "build:esm": "tsc",
-    "build:cjs": "tsc --module commonjs --outDir lib/cjs",
-    "prepare": "npm run build"
-  },
-  "devDependencies": {
-    "@types/react": "^17.0.45",
-    "@types/react-dom": "^17.0.16",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
-    "typescript": "^4.6.4"
+    "build:esm": "tsc && gulp esm",
+    "build:cjs": "tsc --module commonjs --outDir lib/cjs && gulp cjs",
+    "prepare": "npm run build",
+    "test": "react-scripts test"
   },
   "description": "ConfigCat is a configuration as a service that lets you manage your features and configurations without actually deploying new code.",
   "main": "lib/cjs/index.js",
   "types": "lib/esm/index.d.ts",
   "module": "lib/esm/index.js",
+  "files": [
+    "/lib"
```

i guess you should add some pattern here like this:
```suggestion
    "/lib/**/*"
```

---

t works without this new pattern too. Checked with npm pack.